node/src/chain_spec.rs

```diff
@@ -1,19 +1,27 @@
 use cumulus_primitives_core::ParaId;
-use pop_runtime::{AccountId, AuraId, Signature, EXISTENTIAL_DEPOSIT};
+use pop_runtime_common::{AccountId, AuraId, Signature};
 use sc_chain_spec::{ChainSpecExtension, ChainSpecGroup};
 use sc_service::ChainType;
 use serde::{Deserialize, Serialize};
-use sp_core::{sr25519, Pair, Public};
+use sp_core::{crypto::Ss58Codec, sr25519, Pair, Public};
 use sp_runtime::traits::{IdentifyAccount, Verify};
 
-/// Specialized `ChainSpec` for the normal parachain runtime.
-pub type ChainSpec = sc_service::GenericChainSpec<(), Extensions>;
+/// Specialized `ChainSpec` for the development parachain runtime.
+pub type DevnetChainSpec =
+	sc_service::GenericChainSpec<pop_runtime_devnet::RuntimeGenesisConfig, Extensions>;
+
+/// Specialized `ChainSpec` for the testnet parachain runtime.
+pub type TestnetChainSpec =
+	sc_service::GenericChainSpec<pop_runtime_testnet::RuntimeGenesisConfig, Extensions>;
 
 /// The default XCM version to set in genesis config.
 const SAFE_XCM_VERSION: u32 = xcm::prelude::XCM_VERSION;
-
-/// The local para ID of Pop Network.
-const PARA_ID: u32 = 9090;
+pub(crate) enum Relay {
+	ROC,
```

Comment:
Can we use the full network names here instead please? i.e. Rococo, Paseo, RococoLocal and PaseoLocal.

Reply:
Clippy complained about capitalised variant names so used title case instead, addressed in a PR to this one.